docs/en/advanced-concepts/settings.md

```diff
@@ -58,19 +58,24 @@ Settings are implemented via the following API endpoints:
-|`wakeLockTimeout`| Allows to change the timeout of the acquired wake lock. The lock is acquired on server startup and is held until the UIAutomator2 server is killed or the timeout expires. Setting this value to zero or a negative number will release the lock immediately if it is held. Defaults to '24 * 60 * 60 * 1000' milliseconds. | e.g. `0`, `60000` (1 min) |
-|`enableMultiWindows`| If enabled then the page source and xpath lookup will include all windows available to the accessibility rather than the active one only. It might help to resolve finding elements on windows like `android.widget.PopupWindow`. Defaults to `false` and returns an active root window. Read [appium-uiautomator2-server#301](https://github.com/appium/appium-uiautomator2-server/pull/301) for mroe details. |`true` or `false` |
-
+| `actionAcknowledgmentTimeout` | Int (milliseconds) which is the same as [setActionAcknowledgmentTimeout](https://developer.android.com/reference/android/support/test/uiautomator/Configurator.html#setActionAcknowledgmentTimeout(long)). If a negative value is given, it would set to default(3 * 1000 milliseconds). Handled by [UiAutomator Configurator](https://developer.android.com/reference/android/support/test/uiautomator/Configurator.html) in Android API 18 and above. | e.g., `5000` |
+| `allowInvisibleElements` | Boolean Sets whether Android device should show all elements, visible and invisible. `false` by default. | `false` or `true` |
+| `enableMultiWindows` | Boolean If enabled then the page source and xpath lookup will include all windows available to the accessibility rather than the active one only. It might help to resolve finding elements on windows like `android.widget.PopupWindow`. Defaults to `false` and returns an active root window. Read [appium-uiautomator2-server#301](https://github.com/appium/appium-uiautomator2-server/pull/301) for mroe details. |`true` or `false` |
+| `enableNotificationListener` | Boolean which sets whether the Android device should enable or disable the `NotificationListener`. `true` by default.  | `false` or `true` |
+| `keyInjectionDelay` | Int (milliseconds) which is the same as [setKeyInjectionDelay](https://developer.android.com/reference/android/support/test/uiautomator/Configurator.html#setKeyInjectionDelay(long)). If a negative value is given, it would set to default(0 milliseconds). Handled by [UiAutomator Configurator](https://developer.android.com/reference/android/support/test/uiautomator/Configurator.html) in Android API 18 and above. | e.g., `5000` |
+| `mjpegBilinearFiltering` | Int Controls whether or not to apply bilinear filtering to source screenshot resize algorithm. Enabling this flag may improve the quality of the resulting scaled bitmap, but will introduce a [small] performance hit. `false` by default | `false` or `true` |
```

Comment:
What is `Int`?

Reply:
I'd rather remove this prefix everywhere. It does not help to understand the purpose of the setting, but rather makes more confusion

Reply:
I think that's fine by me! Updating!

Reply:
This is resolved in the latest update.